projects/m3/004-string-edit-distance/solutions/enricozoboli/solution_string_edit_distance.py

```diff
@@ -0,0 +1,35 @@
+def edit_distance(s,t):
+    """
+    Takes two  string as a parameter and return the
+    edit distance wich is a measure of similiraty.
+    It is computed with the minimun number of operations
+    (insertion, deletion, substitution)needed to transfom 
+    one string into the other.
+    """
+    
+    if len(s) < 1 :
+        return len(t)
+    elif len(t) < 1:
+        return len(s)
+    else:
+        cost = 0
+        if s[-1] != t[-1]:
+            cost = 1
+        d1 = edit_distance(s[:-1], t) + 1
+        d2 = edit_distance(s, t[:-1]) + 1
+        d3 = edit_distance(s[:-1], t[:-1]) + cost
+        return min(d1, d2, d3) 
+
+
+def main():
+    string_1 = input("Enter the first word: ")
```

Comment:
Since you are pretty confident with coding, you should start to apply a better naming convention to you variables, they should be self explanatory